task/04-arrays-tasks.js

```diff
@@ -234,7 +247,15 @@ function toArrayOfSquares(arr) {
  *   [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ] => [ 1, 3, 6, 10, 15, 21, 28, 36, 45, 55 ]
  */
 function getMovingSum(arr) {
-   throw new Error('Not implemented');
+   let newArr = arr.slice();
+   newArr = newArr.map((value, index, arr) => {
+      if (index > 0) {
+         arr[index] += arr[index - 1];
```

Comment:
Never change arguments. It is a potential bug. Why? If you change the `arr` variable here you will change it in external code.